ipn/ipnlocal/local.go

```diff
@@ -233,7 +241,7 @@ type LocalBackend struct {
 	loginFlags       controlclient.LoginFlags
 	incomingFiles    map[*incomingFile]bool
 	fileWaiters      set.HandleSet[context.CancelFunc] // of wake-up funcs
-	notifyWatchers   set.HandleSet[chan *ipn.Notify]
+	notifyWatchers   set.HandleSet[*watchSession]
```

Comment:
@maisem, can you confirm we're OK to change this?

Reply:
One other alternative here is to make notfiyWatchers a `map[string]chan *ipn.Notify` where string is sessionID.